### ISSUE TITLE
problem with manage.py loaddata

### ISSUE THREAD

**Issue by hjwp** [wagtail/django-modelcluster#98]
from an app i'm working on

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File ".../lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File ".../lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File ".../lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File ".../lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File ".../lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 69, in handle
    self.loaddata(fixture_labels)
  File ".../lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 109, in loaddata
    self.load_label(fixture_label)
  File ".../lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 175, in load_label
    obj.save(using=self.using)
  File ".../lib/python3.6/site-packages/django/core/serializers/base.py", line 208, in save
    getattr(self.object, accessor_name).set(object_list)
  File ".../lib/python3.6/site-packages/modelcluster/fields.py", line 406, in set
    sort_by_fields(objs, rel_model._meta.ordering)
  File ".../lib/python3.6/site-packages/modelcluster/utils.py", line 19, in sort_by_fields
    items.sort(key=lambda x: (getattr(x, key, None) is not None, getattr(x, key)), reverse=reverse)
  File ".../lib/python3.6/site-packages/modelcluster/utils.py", line 19, in <lambda>
    items.sort(key=lambda x: (getattr(x, key) is not None, getattr(x, key)), reverse=reverse)
AttributeError: Problem installing fixture '.../prod.dump.json': 'int' object has no attribute 'title'
```

will try and do a minimal repro if i get a spare mo.

**Comment by NickdeDycker** [comment]
This is because `DeferringManyRelatedManager.set()` is expecting a list of objects, whereas loaddata will pass it a list of integers. This normally doesn't fail, only when your Model has a `Meta.ordering`, where it tries to reorder the objects. When it does so, it loops over your list of what are supposed to be objects and tries `getattr(obj, ORDER_FIELD)`. This raises your attribute error when it's just an integer.

Changing modelcluster.utils:19 to `items.sort(key=lambda x: (getattr(x, key) is not None, getattr(x, key, None)), reverse=reverse)` will solve it without breaking other functionality. Another solution would be to change modelcluster.fields:402 to only allow objects and dictionaries, which might be cleaner. (As the first one will also not raise AttributeError when you actually pass a wrong field).

**Comment by gasman** [comment]
This is hopefully fixed by #118, although in testing I wasn't able to replicate exactly as described here (the test case in #118 fails with `'int' object has no attribute 'id'` even without a Meta ordering) - so if still no luck, please provide a failing test case and I'll reopen.
